ubuntu/common/hpc-tuning.sh

```diff
@@ -30,7 +30,37 @@ cat << EOF >> /etc/security/limits.conf
 EOF
 
 echo "vm.zone_reclaim_mode = 1" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh1 = 4096" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh2 = 8192" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh3 = 16384" >> /etc/sysctl.conf
+echo "sunrpc.tcp_max_slot_table_entries = 128" >> /etc/sysctl.conf
+
+
+## Systemd service for starting sunrpc and adding setting parameters
+cat <<EOF >/usr/sbin/sunrpc_tcp_settings.sh
+#!/bin/bash
+
+modprobe sunrpc
```

Comment:
Does module sunrpc have any dependencies, which need to be loaded first?

Reply:
I don't believe so. "modprobe --show-depends sunrpc" just shows: 

sunrpc.ko